modules/graphdb/templates/00_wait_node_count.sh.tpl

```diff
@@ -11,6 +11,11 @@ set -o errexit
 set -o nounset
 set -o pipefail
 
+# Function to print messages with timestamps
+log_with_timestamp() {
```

Comment:
As a future improvement, you could add `log_info`, `log_error` etc.

Reply:
Noted